core/src/edu/cornell/gdiac/temporary/entity/Note.java

```diff
@@ -1,16 +1,9 @@
 /*
  * Note.java
  *
- * This is a passive model, and this model does very little by itself.  
- * All of its work is done by the CollisionController or the 
- * GameplayController.  
  *
- * This separation is very important for this class because it has a lot 
- * of interactions with other classes.  When a shell dies, it emits stars.  
- * If did not move that behavior to the CollisionController,
- * then we would have to have a reference to the GameEngine in this
- * class.  Tight coupling with the GameEngine is a very bad idea, so
- * we have separated this out.
+ * This separation is very important for this class because it has a lot
```

Comment:
note not shell (again not really important ig)

Reply:
fixed comments